pyproject.toml

```diff
@@ -1,4 +1,81 @@
+[build-system]
+requires = ['setuptools>=51.0.0', 'wheel']
+build-backend = 'setuptools.build_meta'
+
+[project]
+name = 'rq'
+version = '1.13.0'
+authors = [{name = 'Vincent Driessen', email = 'vincent@3rdcloud.com'}]
+maintainers = []
+description = 'RQ is a simple, lightweight, library for creating background jobs, and processing them.'
+readme = 'README.md'
+license = {file = 'LICENSE'}
+requires-python = '>=3.6.2'
+classifiers = [
+    # As from http://pypi.python.org/pypi?%3Aaction=list_classifiers
+    # 'Development Status :: 1 - Planning',
+    # 'Development Status :: 2 - Pre-Alpha',
+    # 'Development Status :: 3 - Alpha',
+    # 'Development Status :: 4 - Beta',
+    'Development Status :: 5 - Production/Stable',
+    # 'Development Status :: 6 - Mature',
+    # 'Development Status :: 7 - Inactive',
+    'Intended Audience :: Developers',
+    'Intended Audience :: End Users/Desktop',
+    'Intended Audience :: Information Technology',
+    'Intended Audience :: Science/Research',
+    'Intended Audience :: System Administrators',
+    'License :: OSI Approved :: BSD License',
+    'Operating System :: POSIX',
+    'Operating System :: MacOS',
+    'Operating System :: Unix',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3 :: Only',
+    'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+    'Topic :: Internet',
+    'Topic :: Scientific/Engineering',
+    'Topic :: System :: Distributed Computing',
+    'Topic :: System :: Systems Administration',
+    'Topic :: System :: Monitoring',
+]
+dependencies = [
+    'redis>=3.0.0,<4',
+    'click>=3.0,<4',
```

Comment:
This also needs to be updated, `requirements.txt` specifies that we require `click>=5.0.0`.

Reply:
fixed